### PR TITLE
Release notes for 0.3.1 and 0.4

### PR DESCRIPTION
**Pull request by darosior**

### CODE REVIEW

**Comment by darosior** [comment]
Did i miss anything?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+#### GUI-specific
+
+- You can now re-register the descriptor on a hardware signing device in the settings.
+- You can now change the alias of each of the signers from the settings panel.
+- You can now connect to a running daemon without having to provide a path to its configuration,
+  provided it is using the default location for its data directory.
+- The GUI will now log to a `installer.log` file at the root of the datadir during installation, and
+  to a `<network>/liana-gui.log` when running. In case of crash, this will contain a backtrace.
+- At signing time we warn you if the descriptor is not registered on the signing device.
+- During installation we check the connection to bitcoind.
```
Comment (edouardparis) [inline CHANGELOG.md:24]:
```suggestion
- During installation we check the connection to bitcoind.
- Signer device aliases are now displayed with the fingerprints
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+  path is thereby lifted.
+
+#### GUI-specific
+
+- You can now re-register the descriptor on a hardware signing device in the settings.
+- You can now change the alias of each of the signers from the settings panel.
+- At signing time we now warn you if the descriptor is not registered on the signing device.
+- The signer alias is now displayed along with its type when signing.
+- You can now connect to a running daemon without having to provide a path to its configuration,
+  provided it is using the default location for its data directory.
```
Comment (darosior) [inline CHANGELOG.md:22]: Or that you provide the `--datadir` option when running the GUI. I love this new feature.

**Comment by darosior** [comment]
self-ACK df5c4ee85d2f138f180842df6aa09d73c663d51d

**Pull request merged**